### DISCUSSION TITLE
MysqlBackup & MysqlServer 8.0.30

### DISCUSSION THREAD

**Discussion by SR-SG2** [MySqlBackupNET/MySqlBackup.Net#92] [Q&A] [answered]
Hy,
I am using MysqlBacup and it is very powerfull with my MySQL servers 5 or 8.
I just installed MySQL server 8.0.30 but unfortunatelly , MysqlBackup it is not working.
Is it an update coming soon ?
Thanks

**Comment by adriancs2** [comment, marked as answer]
There might be some connection string options that you have missed out.

But anyway, you can use the MySqlConnector version, it has best compatibliity and less connection settings needed for MySQL 8.0.

here: https://www.nuget.org/packages/MySqlBackup.NET.MysqlConnector/
